gradle createApp fails on macOS due to invalid version component

Currently, `gradle createApp` fails on branch `develop`, because the version is set to "3.8.0beta" in `gradle.properties`:

```
$ ./gradlew createApp

> Task :createApp FAILED
Bundler Mac Application Image skipped because of a configuration problem: "Version [3.8.0beta] contains invalid component [0beta]"
Advice to fix: Set a compatible 'app-version' value. Valid versions are one to three integers separated by dots.
Command "/Library/Java/JavaVirtualMachines/openjdk17-temurin/Contents/Home/bin/jpackage" failed with RC 1.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':createApp'.
> Error while creating the .app directory.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 19s
6 actionable tasks: 6 executed
```

--- a/gradle.properties
+++ b/gradle.properties
@@ -9,7 +9,9 @@ name = logisim-evolution
 #
 # * Suffix is optional and MUST be kept empty for **stable** rleases.
 # * For non-stable builds use anything that reflects build state (i.e. "dev", "beta1", "rc1").
-version = 3.8.0dev
+#
+# NOTE: if you use suffix, you MUST separate it from the version number with a hypen.
+version = 3.8.0-dev
 
 # Project website URL (the shorter the better)
 url = https://github.com/logisim-evolution/

